### DISCUSSION TITLE
Does the TextField to Input Migration guide needs an update to consider the Field component?

### DISCUSSION THREAD

**Discussion by Wezea** [microsoft/fluentui#29907] [Q&A]
Hi folks,

I am talking about this piece of code mentioned [here](https://react.fluentui.dev/?path=/docs/concepts-migration-from-v8-components-textfield-to-input-migration--page#basic-migration)

Instead of this:

```html
 <div>
   <Label htmlFor={errorId}>With Error Message</Label>
   <Input id={errorId} aria-describedby={errorMessageId} />
   <Text id={errorMessageId}>Error message</Text>
 </div>
```

wouldn't it be better to suggest something like this:

```html
<Field
    label="With Error Message"
    validationMessage="Error message"
  >
    <Input />
 </Field>
```
I would like to follow the best practice you suggest here, but the docs are a bit confusing.

**Comment by spmonahan** [comment]
I think it would be good to use your suggestion. Mind [creating an issue](https://github.com/microsoft/fluentui/issues/new/choose) for it?
